passwords_neilp/pyproject.toml

```diff
@@ -3,8 +3,8 @@ requires = ["flit_core>=3.2"]
 build-backend = "flit_core.buildapi"
 
 [project]
-name = "password-state-client"
-version = "0.0.2"
+name = "password_state_client"
```

Comment:
Will this name change reflect on PyPi? If so, we should consider consolidating the "password state" text.

Reply:
The one on PyPi is already using `password_state_client`

Our repo was behind the project in PyPi